index.html

```diff
@@ -136,7 +147,7 @@ <h2>Tonic</h2>
                       </div>
                   </div>
     </section>
-    <section class="about-myself">
+    <section class="about-myself" id="#about">
         
```

Comment:
- [ ]  Please I was not redirected to the appropriate section when I clicked on the nav links. Kindly address it. 👍
```suggestion
    <section class="about-myself" id="about">
        
```